addons/xeh/CfgEventHandlers.hpp

```diff
@@ -29,26 +29,33 @@ class Extended_InitPost_EventHandlers {};
 class Extended_AnimChanged_EventHandlers {};
 class Extended_AnimDone_EventHandlers {};
 class Extended_AnimStateChanged_EventHandlers {};
+class Extended_CargoLoaded_EventHandlers {};
+class Extended_CargoUnloaded_EventHandlers {};
 class Extended_ContainerClosed_EventHandlers {};
 class Extended_ContainerOpened_EventHandlers {};
 class Extended_ControlsShifted_EventHandlers {};
 class Extended_Dammaged_EventHandlers {};
+class Extended_Deleted_EventHandlers {};
+class Extended_Disassembled_EventHandlers {};
 class Extended_Engine_EventHandlers {};
 class Extended_EpeContact_EventHandlers {};
 class Extended_EpeContactEnd_EventHandlers {};
 class Extended_EpeContactStart_EventHandlers {};
 class Extended_Explosion_EventHandlers {};
 class Extended_Fired_EventHandlers {};    // Backwards compatibility, uses XEH notation
 class Extended_FiredBIS_EventHandlers {}; // New fired EH, uses BIS notation
+class Extended_FiredMan_EventHandlers {};
 class Extended_FiredNear_EventHandlers {};
 class Extended_Fuel_EventHandlers {};
 class Extended_Gear_EventHandlers {};
+class Extended_GestureChanged_EventHandlers {};
+class Extended_GestureDone_EventHandlers {};
 class Extended_GetIn_EventHandlers {};
 class Extended_GetInMan_EventHandlers {};
 class Extended_GetOut_EventHandlers {};
 class Extended_GetOutMan_EventHandlers {};
-//class Extended_HandleDamage_EventHandlers {}; // Not implemented yet
-//class Extended_HandleHeal_EventHandlers {};   // Not implemented yet
+class Extended_HandleDamage_EventHandlers {};
+class Extended_HandleHeal_EventHandlers {};
```

Comment:
Last time I checked, they ignored the return value. It was better to add handleDamage via init class event.

Does it work now? If not, don't enable these. It will only cause anger and confusion.

Reply:
Should `HandleHeal` be kept in `script_component.hpp` and `script_xeh.hpp`, or removed there as well?

Reply:
Remove it.

If you want support for HandleDamage and HandleHeal in addClassEventHandler:

- make sure the RV is still ignored in current version
- if yes, make addClassEventHandler add a retroactive flag set init event handler that uses the addEventHandler command

Those would be removable by removeAllEventHandlers and removeEventHandler with hard-coded ehids, but anyone who uses the former is a terrorist and the latter goes against best practices so is on their own.

Reply:
I'm worried about adding HandleDamage now
it's going to take a ton of testing to make sure it doesn't cause problems with old mods that may have used this
and to make sure that the return value doesn't override existing code